### PR TITLE
created all polls fe/integration

### PR DESCRIPTION
**Pull request by akhilarjun3-sureify**
Screenshot from 2023-06-28 12-41-12
Screenshot from 2023-06-28 12-41-29
Screenshot from 2023-06-28 12-42-30

### CODE REVIEW

**Review by rameshguptha-sureify — APPROVED** [review]

File: src/components/user/ViewPolls.jsx [hunk 1]
```diff
@@ -19,9 +20,9 @@ export const ViewPolls = ({ activeFlag, pollsData }) => {
                     />
                 ))
             ) : activeFlag ? (
-                <EmptyDataContainer isActive={activeFlag} />
+                <EmptyDataContainer isActive={activeFlag} type={type} />
```
Comment (maniknr-sureify) [inline src/components/user/ViewPolls.jsx:23]: remove isactive flag and just pass the sentence as prop

File: src/components/user/EmptyDataContainer.jsx [hunk 2]
```diff
@@ -1,22 +1,34 @@
...
+        justifyContent: "center",
+        gap: 2,
+      }}
+    >
+      <img
+        src="/assets/notfound.jpeg"
+        alt="404"
+        style={{ width: "150px", borderRadius: "10px" }}
+      />
+      {type === "activePolls" ? (
```
Comment (maniknr-sureify) [inline src/components/user/EmptyDataContainer.jsx:21]: remove this and in typography directly render prop sentence

File: src/components/user/ViewPolls.jsx (hunk 1 above)
Comment (akhilarjun3-sureify) [reply src/components/user/ViewPolls.jsx]: changed

File: src/components/user/EmptyDataContainer.jsx (hunk 2 above)
Comment (akhilarjun3-sureify) [reply src/components/user/EmptyDataContainer.jsx]: changed

**Review by maniknr-sureify — APPROVED** [review]

**Pull request merged**